docs/content/installation/installing-nic/installation-with-helm.md

```diff
@@ -335,6 +335,7 @@ The following tables lists the configurable parameters of the NGINX Ingress Cont
 | **controller.initContainers** | InitContainers for the Ingress Controller pods. | [] |
 | **controller.extraContainers** | Extra (eg. sidecar) containers for the Ingress Controller pods. | [] |
 | **controller.resources** | The resources of the Ingress Controller pods. | requests: cpu=100m,memory=128Mi |
+| **controller.initContainerResources** | The resources of the init container which is used when `controller.readOnlyRootFilesystem` is set to `true` | requests: cpu=100m,memory=128Mi |
```

Comment:
@shaun-nx @haywoodsh Is this the file which gets overwritten by /nginx-ingress/README.md during release?

Reply:
Have removed this change, thanks @vepatel!